news/2017-10-09-halloween-fanart-contest.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "Halloween 2017 Fanart Contest"
+date: 2017-10-09 06:00:00 +0000
+---
+
+Spooks, chills and thrills abound! Halloween is drawing close again, and you know what that means - a fanart contest! Two hectic weeks to pen any of the osu! mascots (or community mascots) in a delightfully spooky Halloween setting, for a chance at getting your work added to the game as a seasonal background!
```

Comment:
`Spooks, chills, and thrills`
put a comma after `chills` but before `and`

Reply:
oxford comma after chills delineates 'thrills' voicing and unpairs it from chills in terms of cadence

that explanation is uh.. let's just say it sounds better (to me) without the comma